newrelicAPI.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
Definitely ignore this file, and make a template file. I.e newRelicAPIExample and add key: "to do". 

**Rationale:** You don't want to expose your keys on GitHub ever for security reasons.